test/libsolidity/syntaxTests/literalSuffixes/suffixableLiterals/invalid_suffixed_address_as_uint.sol

```diff
@@ -4,4 +4,4 @@ contract C {
     uint x = 0x1234567890123456789012345678901234567890 suffix;
 }
 // ----
-// TypeError 8838: (100-149): The type of the literal cannot be converted to the parameter of the suffix function.
+// TypeError 8838: (100-149): The literal cannot be converted to the type accepted by the suffix function.
```

Comment:
This one may not be easy for users to figure out. Would be best if the message could say that it's because this literal is of `address` type.